### PR TITLE
Further improve button grouping

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card

https://trello.com/c/QHKZbml7

### Context

- We know that button grouping is not implemented as it should be based on the design system
- This has been fixed in some areas in preious PRs
- This PR covers the final set

### Changes proposed in this pull request

- Group button and link on the 'change school' dashboard
- Group button and link on CSV export
- Group buttons on our accept/reject request page 

### Guidance to review

For the accept/reject request page, go to this page whilst being logged in as Outwood Academy Acklam: https://get-school-experience-review-pr-3.test.teacherservices.cloud/schools/placement_requests/370

### CODE REVIEW

**Comment by MylesJarvis** [comment]
School switching page

**Before**
image


**After**

image

NB: Font size seems to have increased following grouping - worth looking into given some areas where this grouping has happened, the text size has decreased.

**Comment by MylesJarvis** [comment]
Accept/reject request page

**Before**

image


**After**

image

**Comment by MylesJarvis** [comment]
CSV Export

**Before**

image

**After**

image

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me from a front-end perspective

Needs a tech review as well

File: app/views/schools/change_schools/show.html.erb [hunk 1]
```diff
@@ -29,6 +33,7 @@
         <p class="govuk-body-l">
           You do not have access to any schools yet.
         </p>
+            <%= render 'pages/request_organisation_access' %>
```
Comment (martyn-w) [inline app/views/schools/change_schools/show.html.erb:36]: line 36 looks to be a duplicate of line 38?

Comment (gemmadallmandfe) [reply app/views/schools/change_schools/show.html.erb:36]: Good spot!

@MylesJarvis Will leave this to you to resolve on your return

image

Comment (MylesJarvis) [reply app/views/schools/change_schools/show.html.erb]: Excellent catch, pushed fix

Comment (MylesJarvis) [reply app/views/schools/change_schools/show.html.erb]: image

@martyn-w this is fixed.

**Review by martyn-w — APPROVED** [review]
👍  LGTM

**Pull request merged**